test/unit/bulk-load-test.js

```diff
@@ -1,13 +1,24 @@
 const { assert } = require('chai');
 
 const BulkLoad = require('../../src/bulk-load');
+const TYPES = require('../../src/data-type').typeByName;
 
 describe('BulkLoad', function() {
   it('starts out as not being canceled', function() {
     const request = new BulkLoad('tablename', { tdsVersion: '7_2' }, { });
     assert.strictEqual(request.canceled, false);
   });
 
+  it('add row with a value has a wrong data type', function() {
+    const request = new BulkLoad('tablename', { tdsVersion: '7_2', validateBulkLoadParameters: true }, { });
+    request.addColumn('columnName', TYPES.Date, { nullable: true });
+    try {
+      request.addRow({ columnName: 'Wrong Input' });
+    } catch (err) {
+      assert.strictEqual(err.message, 'Invalid date.');
+    }
```

Comment:
I think this should use the `assert.throws` helper:
```suggestion
    assert.throws(() => {
      request.addRow({ columnName: 'Wrong Input' });
    }, TypeError, 'Invalid date.');
```

Reply:
Oh, sure. There are few other tests in this test file that also using strictEqual, should I also modify them as well?

Reply:
Nvm, those are under "\test\integration\bulk-load-test.js" 😅